DigitalLearningSolutions.Web.Tests/DigitalLearningSolutions.Web.Tests.csproj

```diff
@@ -7,6 +7,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <AdditionalFiles Remove="C:\Users\Sherif.Olaboye\Source\Repos\DLSV2\StyleCop.json" />

```

Comment:
I'm not sure why this has  been added. It seems to reference a path on your local machine.